pages/new-post.js

```diff
@@ -0,0 +1,82 @@
+import MainContext from '../context/MainContext.js'
+import { useState, useContext } from 'react';
+import Nav from '../components/navbar/Nav.jsx';
+import { FormControl, FormLabel, Textarea, Input, Select, Button } from "@chakra-ui/react";
+import Confirm from '../components/new-post/Confirm.jsx';
+import Head from 'next/head';
+import axios from 'axios';
+
+const NewPost = () => {
+
+  const [cat, setCat] = useState(null);
+  const handleCat = (e) => {
+    setCat(e.target.value);
+    console.log(e.target.value);
+  };
+  const [top, setTop] = useState(null);
+  const handleTop = (e) => {
+    setTop(e.target.value);
+    console.log(e.target.value);
+  };
+  const [txt, setTxt] = useState(null);
+  const handleTxt = (e) => {
+    setTxt(e.target.value);
+    console.log(e.target.value);
+  };
+
+  const info = {
+    category: cat || '',
+    email: 'pillsbury.doughboy@gmail.com',
+    body: txt || '',
+    title: top || ''
+  };
+
+  const handleSubmit = () => {
+    event.preventDefault();
+    // console.log(info);
+    axios.post('/api/createPost', info)
+  }
+
+
+  // const { handleSubmit } = useContext(MainContext);
+  const allcat = ['Sports', 'Food', 'Politics', 'Arts & Crafts', 'Film & TV', 'Video Games', 'Fashion', 'Technology', 'Travel', 'Automotive', 'Celebrity', 'Music', 'Science & Nature', 'Religion'];
```

Comment:
This is good for now. After the feature freeze we should have the Categories in the Context Provider that we can all use to keep it uniform.